packages/contact-center/user-state/src/helper.ts

```diff
@@ -1,11 +1,44 @@
-export const useUserState = () => {
+import {useState, useEffect} from "react";
 
+export const useUserState = ({idleCodes, agentId, cc}) => {
 
-  const handleAgentStatus = (event: { target: { value: string; }; }) => {
-  };
+  const [isSettingAgentStatus, setIsSettingAgentStatus] = useState(false);
+  const [errorMessage, setErrorMessage] = useState('');
+
+  const [elapsedTime, setElapsedTime] = useState(0);
+
+  useEffect(() => {
+    // Reset the timer whenever the component mounts or the state changes
+    setElapsedTime(0);
+    const timer = setInterval(() => {
+      setElapsedTime(prevTime => prevTime + 1);
+    }, 1000);
```

Comment:
On WC, the timer starts when the component is already in the DOM making it not a 0 seconds timer when appearing. Needs to be fixed

Reply:
Fixed this